### ISSUE TITLE
do not work with archisteamfarm 6.0.5.2

### ISSUE THREAD

**Issue by feeops** [Citrinate/CS2Interface#19]
do not work with archisteamfarm 6.0.5.2

curl "xxx/Inventory?minimal=false&showDefs=false"

return is empty

**Comment by Citrinate** [comment]
I was able to discover and fix an error on non-generic ASF V6.0.5.2 that prevented the plugin from loading a bot's inventory:

```
2024-08-07 20:21:51|ArchiSteamFarm-2946884|ERROR|bot|HandleCallbacks() System.MissingMethodException: Method not found: 'Single System.BitConverter.ToSingle(Byte[], Int32)'.
   at CS2Interface.InventoryItem.ParseAttributes(List`1 attributes)
   at CS2Interface.InventoryItem.SetAdditionalProperties()
   at CS2Interface.InventoryItem..ctor(CSOEconItem item)
   at CS2Interface.Client.<>c.<OnClientWelcome>b__19_0(Byte[] x)
   at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
   at System.Collections.Concurrent.ConcurrentDictionary`2.InitializeFromCollection(IEnumerable`1)
   at System.Collections.Concurrent.ConcurrentDictionary`2..ctor(Int32, IEnumerable`1 , IEqualityComparer`1 )
   at System.Collections.Concurrent.ConcurrentDictionary`2..ctor(IEnumerable`1 )
   at CS2Interface.Client.OnClientWelcome(IPacketGCMsg packetMsg)
   at CS2Interface.Client.OnGCMessage(MessageCallback callback)
   at SteamKit2.Internal.Callback`1.Run(Object callback)
   at SteamKit2.CallbackManager.RunWaitAllCallbacks(TimeSpan timeout)
   at ArchiSteamFarm.Steam.Bot.HandleCallbacks()
```

Please download the latest build and check that it fixes your issue: [CS2Interface V1.0.10.2](https://github.com/Citrinate/CS2Interface/actions/runs/10293998492/artifacts/1788274877)

If the issue isn't fixed, then I'll need to see your logs.

**Comment by feeops** [comment]
It works now.
